### DISCUSSION TITLE
Running OpenTopas Pronlem[Run issue]:

### DISCUSSION THREAD

**Discussion by datboigbenga** [OpenTOPAS/OpenTOPAS#81] [Run-Problem]
### Contact Details

eruolagbenga@gmail.com

### What was the issue?

I tried running OpenTopas but i keep getting error involving libQt5OpenGL.5.dylib

### TOPAS category

Geometry

### Version

4.0

### What Operative System (OS) are you running on?

MacOS

### Relevant log output

```shell
dyld[33518]: Library not loaded: @rpath/libQt5OpenGL.5.dylib
  Referenced from: <024B023A-685D-33EB-A31B-459617E81516> /Applications/TOPAS/OpenTOPAS-install/bin/topas
  Reason: tried: '/Applications/GEANT4/geant4-install/lib/libQt5OpenGL.5.dylib' (no such file), '/Applications/TOPAS/OpenTOPAS-install/lib/libQt5OpenGL.5.dylib' (no such file), '/libQt5OpenGL.5.dylib' (no such file), '/Applications/TOPAS/OpenTOPAS-install/Frameworks/libQt5OpenGL.5.dylib' (no such file), '/Applications/TOPAS/OpenTOPAS-install/Frameworks/libQt5OpenGL.5.dylib' (no such file)
/Users/apple/shellScripts/topas: line 7: 33518 Abort trap: 6           /Applications/TOPAS/OpenTOPAS-install/bin/topas $1
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment]
Hi,

If you're following the [quickStart guide](https://github.com/OpenTOPAS/OpenTOPAS/blob/main/OpenTOPAS_quickStart_MacOS.md) could you please verify that you're using the output of step 6.1 as the qt path in step 6.2?

All the best,
Thongchai

**Reply by datboigbenga** [reply]
Thank you for the reply, I followed the instruction above but i still  get same error
